scripts/averages.js

```diff
@@ -0,0 +1,44 @@
+var pry = require('pryjs');
+
+function total(array) {
+  var sum = array.reduce(function (total, number) {
+    return total + number;
+  }, 0);
+return sum
+};
+
+function mean(array) {
+  var total = array.reduce(function (total, number) {
```

Comment:
@case-eee here's where we tried to call total(array) but it didn't like it at all, so instead the code has been duplicated. Any thoughts?